test/unittests/serviceRoutines/putIndividualContextEntityAttribute_test.cpp

```diff
@@ -81,9 +81,9 @@ TEST(putIndividualContextEntityAttribute, json)
   ci.payloadSize    = strlen(testBuf);
 
   serviceVectorsSet(NULL, putV, NULL, NULL, NULL, NULL, badVerbV);
-  out = orion::requestServe(&ci);
+/*  out = orion::requestServe(&ci);
```

Comment:
This seems to be pending... only the present() calls should be removed

Reply:
fixed in 5dc2fa8